packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -392,7 +395,7 @@ export const UIModeView: React.FC<{}> = ({
           <div className='section-title'>Playwright</div>
           <ToolbarButton icon='color-mode' title='Toggle color mode' onClick={() => toggleTheme()} />
           <ToolbarButton icon='refresh' title='Reload' onClick={() => reloadTests()} disabled={isRunningTest || isLoading}></ToolbarButton>
-          <ToolbarButton icon='terminal' title='Toggle output' toggled={isShowingOutput} onClick={() => { setIsShowingOutput(!isShowingOutput); }} />
+          <ToolbarButton icon='terminal' title={'Toggle output - ' + (navigator.platform === 'MacIntel' ? '⌘ + `' : 'Ctrl + `')} toggled={isShowingOutput} onClick={() => { setIsShowingOutput(!isShowingOutput); }} />
```

Comment:
```suggestion
          <ToolbarButton icon='terminal' title={'Toggle output — ' + (navigator.platform === 'MacIntel' ? '⌃ `' : 'Ctrl + `')} toggled={isShowingOutput} onClick={() => { setIsShowingOutput(!isShowingOutput); }} />
```

Reply:
I tried to align it with Chromium wdyt? https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/front_end/panels/recorder/RecorderController.ts;l=1055?q=filepath:devtools-frontend%20%22%E2%8C%98%22%20-filepath:out%2F&ss=chromium%2Fchromium%2Fsrc